### ISSUE TITLE
Support displaying multiple currencies 

### ISSUE THREAD

**Issue by baimafeima** [Eqonomize/Eqonomize#28]
Do you currently support displaying multiple currencies at the same time? More specifically, I would like to be able list my Chinese, British and German bank accounts and view them simultaneously in their native currencies.

**Comment by hanna-kn** [comment]
Yes. You can specify a different currency for each account. The value of each separate account and transaction are displayed in the associated currency.

**Comment by baimafeima** [comment]
I see, great! Is there a possibility to make them "comparable" to one another by showing all of them in a predefined base currency so that the total available assets are displayed in one currency?

**Comment by hanna-kn** [comment]
The total assets in the accounts view are converted to the selected main currency. To compare the value of separate accounts you can currently use the "categories comparison chart", which includes an option to display all accounts in a pie or bar chart.
